apps/include/apps.h

```diff
@@ -37,6 +33,17 @@
 # include "fmt.h"
 # include "platform.h"
 # include "engine_loader.h"
+# include "ca.h"
+
+#ifndef F_OK
```

Comment:
These should not be needed if the test is just a normal test app.

Reply:
I agree concerning [FRW]_OK - defines. The include for `ca.h` into `apps.h` is required for DB_ATTR and CA_DB.